dashboard/test/helpers/delete_accounts_helper_test.rb

```diff
@@ -775,6 +775,79 @@ def run(*_args, &_block)
     assert_nil application.notes
   end
 
+  #
+  # Table: dashboard.pd_facilitator_program_registrations
+  #
+
+  test "clears form_data from pd_facilitator_program_registrations" do
+    registration = create :pd_facilitator_program_registration
+    refute_equal '{}', registration.form_data
+
+    purge_user registration.user
+
+    registration.reload
+    assert_equal '{}', registration.form_data
+  end
+
+  #
+  # Table: dashboard.pd_fit_weekend1819_registrations
+  # Associated with user via application
+  #
+
+  test "clears form_data from pd_fit_weekend1819_registrations" do
+    registration = create :pd_fit_weekend1819_registration
+    refute_equal '{}', registration.form_data
+
+    purge_user registration.pd_application.user
+
+    registration.reload
+    assert_equal '{}', registration.form_data
+  end
+
+  #
+  # Table: dashboard.pd_pre_workshop_surveys
+  # Associated with user via enrollment
+  #
+
+  test "clears form_data from pd_pre_workshop_surveys" do
+    Pd::Workshop.any_instance.stubs(:pre_survey_units_and_lessons).returns([])
+
+    enrollment = create :pd_enrollment, :from_user
+    survey = create :pd_pre_workshop_survey,
+      pd_enrollment: enrollment,
+      form_data_hash: {unit: Pd::PreWorkshopSurvey::UNIT_NOT_STARTED}
+    refute_equal '{}', survey.form_data
+
+    purge_user survey.pd_enrollment.user
+
+    survey.reload
+    assert_equal '{}', survey.form_data
+  end
+
+  #
+  # Table: dashboard.pd_regional_partner_program_registrations
+  #
```

Comment:
I'm pretty sure this can be associated with regular PD'd teachers, not only with regional partners, so we need to delete it.  It also won't let us null out the user_id and includes a `teachercon` column that (I think) would associate that user id with a particular teachercon.  This is potentially de-anonymizing information, so I've gone ahead and set the teachercon number to an invalid value as well.